test/e2e/windows/hybrid_network.go

```diff
@@ -37,7 +38,7 @@ const (
 
 var (
 	windowsBusyBoximage = imageutils.GetE2EImage(imageutils.Agnhost)
-	linuxBusyBoxImage   = imageutils.GetE2EImage(imageutils.Nginx)
+	linuxBusyBoxImage   = imageutils.GetE2EImage(imageutils.BusyBox)
```

Comment:
I understand why it fails now, the busybox image requires to set a `sleep` command or the containers crash loops

Reply:
bring back the previous image by now, I'll figure it out later